src/main/java/com/flippingutilities/model/HistoryManager.java

```diff
@@ -360,52 +342,67 @@ public void validateGeProperties()
 
 	public boolean hasValidOffers()
 	{
-		return compressedOfferEvents.stream().anyMatch(OfferEvent::isValidOfferEvent);
+		return compressedOfferEvents.size() > 0;
```

Comment:
Yea, essentially the validOfferEvent field has been useless as we delete offers as soon as we mark them invalid, instead of marking them as invalid and then pruning them later. That was a result of some legacy code for reasons I'm not sure of. 

I've now deleted that field entirely